api-report/aqueduct.api.md

```diff
@@ -148,8 +148,8 @@ export abstract class PureDataObject<I extends DataObjectTypes = DataObjectTypes
     // (undocumented)
     get disposed(): boolean;
     finishInitialization(existing: boolean): Promise<void>;
-    // (undocumented)
-    static getDataObject(runtime: IFluidDataStoreRuntime): Promise<PureDataObject<DataObjectTypes>>;
+    // @deprecated (undocumented)
+    static getDataObject(runtime: IFluidDataStoreRuntime): Promise<PureDataObject<any>>;
```

Comment:
i wouldn't expect this change

Reply:
Agree, neither would I, I didn't spot it yesterday. Fixed locally, will disappear with the next push.